tests/unit/layers/test_layers_losses.py

```diff
@@ -0,0 +1,211 @@
+"""Tests for layers.bpr"""
+
+import numpy as np
+import tensorflow as tf
+
+import deepr as dpr
+
+
+def test_layers_bpr():
```

Comment:
I see that you made on file with all the losses -> hence one file for test.
maybe more readable to split each class in its own file but just a nit

Reply:
Yes, why not

Reply:
splitted